.drone.env

```diff
@@ -1,5 +1,5 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=35c27afcb6a086ab0e393f356cfd4b005840b146
+OCIS_COMMITID=47c2c3ad4f0990ffa6456777cfd46af090bd4120
```

Comment:
oCIS PR https://github.com/owncloud/ocis/pull/2477 has been merged to update reva in oCIS.

Should the `OCIS_COMMITID` be bumped again here to `0e9e6356727590bc8e18d20dbe0425903f24bece` to make sure that web works fine with the changed oCIS?

Reply:
Thanks for the hint, done!